src/components/FileReader.vue

```diff
@@ -0,0 +1,68 @@
+<script lang="ts" setup>
+import { ref, reactive, computed } from 'vue'
+import { useImportStore } from '@/stores/import'
+import type MatlabFile from "@/interfaces/IExpectedFileContent"
+
+const docFile: any = ref(null)
+const importFile: any = useImportStore()
+
+const file: any = reactive({})
+const content: any = ref("")
+const loading: any = ref(false)
+
+function readMatlabTxtFile(): void {
+    file.value = docFile.value.files[0];
+    const reader = new FileReader();
+    if (file.value.name.includes(".txt")) {
+        reader.onload = (res) => {
+            content.value = res.target!.result;
+        };
+        reader.readAsText(file.value);
+    }
+}
+
+function parseMatLabTxtData(): void {
```

Comment:
User is unable to not *quite follow the proper formatting* since it is handled by the MATLAB export functionality which always stays the same. This method is solely responsible for handling `.txt` file imports and thus far, these are the constrinats expected therefore this method is expected to work with any import of a `.txt` file generated by MATLAB. It would be very expensive to check every line's formatting to see if there are any invalid lines so I think it is best we leave it how it is and another user story can handle exceptions for invalid `.txt` files that can't be parsed.